content/features/featuresDeepDive/importers/glTF/createExtensions.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Create glTF extensions
+image:
+description: Learn about creating new glTF loader extensions.
+keywords: diving deeper, import, importing assets, asset, glTF, extensions
+further-reading:
+video-overview:
+video-content:
+---
+
+## Introduction

```

Comment:
Maybe link to [my blog](https://babylonjs.medium.com/extending-the-gltf-loader-in-babylon-js-588e48fb692b) somewhere in this doc.